src/TwigJs/JsCompiler.php

```diff
@@ -180,7 +180,73 @@ public function __construct(\Twig_Environment $env)
             'Twig_Node_Expression_Test_Null' => new Compiler\Expression\Test\NullCompiler(),
             'Twig_Node_Expression_Test_Odd' => new Compiler\Expression\Test\OddCompiler(),
             'Twig_Node_Expression_Test_Sameas' => new Compiler\Expression\Test\SameasCompiler(),
-
+            \Twig\Node\Node::class => new NodeCompiler(),
+            \Twig\Node\BodyNode::class => new BodyCompiler(),
+            \Twig\Node\ModuleNode::class => new ModuleCompiler\GoogleCompiler(),
+            \Twig\Node\BlockNode::class => new BlockCompiler(),
+            \Twig\Node\TextNode::class => new TextCompiler(),
+            \Twig\Node\IfNode::class => new IfCompiler(),
+            \Twig\Node\PrintNode::class => new PrintCompiler(),
+            \Twig\Node\ForNode::class => new ForCompiler(),
+            \Twig\Node\ForLoopNode::class => new ForLoopCompiler(),
+            \Twig\Node\SetNode::class => new SetCompiler(),
+            \Twig\Node\IncludeNode::class => new IncludeCompiler(),
+            \Twig\Node\SpacelessNode::class => new SpacelessCompiler(),
+            \Twig\Node\SetTempNode::class => new SetTempCompiler(),
+            \Twig_Node_Expression_ExtensionReference::class => new ExtensionReferenceCompiler(),
```

Comment:
This class is deprecated and has no namespace variant (also, the string based mapping is wrong).